samples/bluetooth/throughput/README.rst

```diff
@@ -124,8 +125,8 @@ For the tester::
    [bt] [INF] bt_dev_show_info: HCI: version 5.0 (0x09) revision 0x0000, manufacturer 0x05f1
    [bt] [INF] bt_dev_show_info: LMP: version 5.0 (0x09) subver 0xffff
    Bluetooth initialized
-   Advertising successfully started
-   Scanning successfully started
+   Choose device role write s (slave role) or m (master role):
```

Comment:
Don't forget to update this log if you follow @migr suggestion:

https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2117/files#r407971794

Reply:
Ok